_includes/templates/compatibility-page.md

```diff
@@ -1,3 +1,5 @@
+![{{tool.name|escape_once}}]({{tool.logo}}){:.third-party-logo}
```

Comment:
Can you add a title attribute to the img, so that there's a tooltip when you hover over it?